tests/test_fuzzing.py

```diff
@@ -227,6 +251,8 @@ def test_fuzzing_hangs(pcs_full):
 
     hang_test = os.path.join(examples,  "hang-test", "hang")
 
+    # Fixme: This test is shaky, and should be implemented in different way; it can sometimes fail with error
+    # during the initial testing
```

Comment:
I think this should be extracted into a separate issue so that we know this needs our attention (tests that fail nondeterministically are really annoying).